docs/installation.rst

```diff
@@ -51,6 +51,23 @@ There are several methods that can be used to install HistomicsTK.  Each of thes
 .. include:: ../ansible/README.rst
    :start-after: __methods
 
+Installing Analysis Images
```

Comment:
Maybe rename to something along the lines of `Installing Docker Images containerizing Slicer CLIs for Analysis` to be clear.